flixel/addons/editors/tiled/TiledMap.hx

```diff
@@ -114,6 +120,13 @@ class TiledMap
 	{
 		for (node in source.nodes.tileset)
 		{
+			if (node.has.source)
+			{
+				var path = '${rootPath}/${node.att.source}';
+				path = REDUNDANT_PATH.replace(path, "/");
```

Comment:
Is `REDUNDANT_PATH` trying to do the same as `Path.normalize()` (which is used in https://github.com/HaxeFlixel/flixel-addons/pull/312)?